error_checks.c

```diff
@@ -101,11 +101,3 @@ bool check_valve_pin_error(enum VALVE_STATE req_state) {
     // check that digital pin reads match what they should be (a la injector)
```

Comment:
bool check_valve_pin_error(enum VALVE_STATE req_state) just returns true right now, at absolute minimum a TODO comment should be added, preferably just implement it.

Reply:
adding TODO for now